source/app.js

```diff
@@ -1,91 +1,97 @@
-(function () {
-  var options = INSTALL_OPTIONS
-  var id
-
+(function() {
+  var options = INSTALL_OPTIONS;
+  var id;
 
   if (options.account && options.organization) {
-    id = options['web-properties-for-' + options.organization]
+    id = options["web-properties-for-" + options.organization];
   } else {
-    id = (options.id || '').trim()
+    id = (options.id || "").trim();
   }
 
   if (!id) {
-    console.log('Cloudflare Google Analytics: Disabled. UA-ID not present.')
-    return
-  } else if (INSTALL_ID === 'preview') {
-    console.log('Cloudflare Google Analytics: Enabled', id)
+    console.log("Cloudflare Google Analytics: Disabled. UA-ID not present.");
+    return;
+  } else if (INSTALL_ID === "preview") {
+    console.log("Cloudflare Google Analytics: Enabled", id);
   }
 
-  function resolveParameter (uri, parameter) {
+  function resolveParameter(uri, parameter) {
     if (uri) {
-      var parameters = uri.split('#')[0].match(/[^?=&]+=([^&]*)?/g)
+      var parameters = uri.split("#")[0].match(/[^?=&]+=([^&]*)?/g);
 
-      for (var i = 0, chunk; chunk = parameters[i]; ++i) {
+      for (var i = 0, chunk; (chunk = parameters[i]); ++i) {
         if (chunk.indexOf(parameter) === 0) {
-          return unescape(chunk.split('=')[1])
+          return unescape(chunk.split("=")[1]);
         }
       }
     }
   }
 
-  window.dataLayer = window.dataLayer || []
-  function gtag () { window.dataLayer.push(arguments) }
+  window.dataLayer = window.dataLayer || [];
+  function gtag() {
+    window.dataLayer.push(arguments);
+  }
 
-  gtag('js', new Date())
-  gtag('config', id)
+  gtag("js", new Date());
+  gtag("config", id);
+  gtag("set", { anonymizeIp: options.anonymizeIp });
```

Comment:
Looks like prettier had its way with this file.  The only relevant change is here.